Essentials/src/com/earth2me/essentials/signs/SignDisposal.java

```diff
@@ -12,8 +14,21 @@ public SignDisposal() {
     }
 
     @Override
```

Comment:
Please see SignBuy - you are missing the onSignCreate hook to validate / fixup the sign. You only need to validate the final line - not lines 1/2.

Reply:
This performs validation on the price when the sign is created as well as fixes up the price into the right format

Reply:
Ok, thanks. I'm pretty new to Essentials sources, that's why I'm missing stuff like this... Will resolve this shortly.